audioset.py

```diff
@@ -461,14 +458,15 @@ def _openAudSet(this, fname):
 class ClipDownloader(object):
     ''' downloader for youtube clips '''
     
-    def __init__(this, data_dir=None, logLvl= logging.WARN):
+    def __init__(this, data_dir=None, sampling_rate = None,logLvl= logging.WARN):
```

Comment:
We might want to rethink this a little better.  I'm worried you are going to end up with a bunch of downsampled clips in the same data directory as non-downsampled clips, and audioset won't know which are which.  Perhaps we want to put in a warning or something if the pre-downloaded clip is a sampling rate less than what you have requested?  Or re-download it?